database/tables/tables_sqlServer.sql

```diff
@@ -1,211 +1,211 @@
 -- this script is for SQL Server and Azure SQL
 
-USE [enter_db_name_here]
+USE [enter_db_name_here];
 GO
 
-IF EXISTS (SELECT * FROM dbo.sysobjects WHERE id = OBJECT_ID(N'[dbo].[FK_QRTZ_TRIGGERS_QRTZ_JOB_DETAILS]') AND OBJECTPROPERTY(id, N'ISFOREIGNKEY') = 1)
```

Comment:
duo.sysobject is deprecated (https://docs.microsoft.com/en-us/sql/relational-databases/system-compatibility-views/sys-sysobjects-transact-sql)